### PR TITLE
Send notification when a users verification status changes

### PR DESCRIPTION
**Pull request by doshitan**
Issue #250 

@krues8dr @sethetter 

This has some interaction with things that change in #968, so whichever one gets merged first, the other will need to be rebased before it can be merged.

### CODE REVIEW

File: app/Listeners/UserVerificationStatusChangeNotification.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    /**
+     * Handle the event.
+     *
+     * @param  UserVerificationStatusChange  $event
+     * @return void
+     */
+    public function handle(UserVerificationStatusChange $event)
+    {
+        $text = "Your account verification status has changed from {$oldValue} to {$newValue}.";
```
Comment (sethetter) [inline app/Listeners/UserVerificationStatusChangeNotification.php]: I'm getting `Undefined variable: oldValue` here. I see that in implements the above class though, should it be using `$this->oldValue`? Is this a PHP version issue?

Comment (sethetter) [inline app/Listeners/UserVerificationStatusChangeNotification.php]: Found it, should have been referencing `$event->oldValue` it looks like.

Comment (doshitan) [inline app/Listeners/UserVerificationStatusChangeNotification.php]: Right, fixed.

**Comment by krues8dr** [comment]
Looks good 👍

**Comment by sethetter** [comment]
Saw the change, that was the only thing I caught, so I'll give it a 👍 !

**Pull request merged**